shop.json

```diff
@@ -0,0 +1,8 @@
+{
+  "items": [
+    {"name": "Dagger", "description": "Kill one player", "price": "750", "emoji": ":dagger:"},
+    {"name": "Randium Face Reveal", "description": "Randium will send a picture of his face", "price": "9999999999", "emoji": ":grinning_face_with_big_eyes:"}
+  ],
+  "currency": "WBucks",
```

Comment:
Credits. Werewolf bucks are something different.

Reply:
That's why it's in the config, you can change it easily :)